azure-iot-device/samples/sync-samples/recurring_telemetry.py

```diff
@@ -0,0 +1,48 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+import uuid
+import time
+from azure.iot.device import IoTHubDeviceClient
+from azure.iot.device import Message
+
+
+def main():
+    # The connection string for a device should never be stored in code. For the sake of simplicity we're using an environment variable here.
+    conn_str = os.getenv("IOTHUB_DEVICE_CONNECTION_STRING")
+    # The client object is used to interact with your Azure IoT hub.
+    device_client = IoTHubDeviceClient.create_from_connection_string(conn_str)
+
+    print("IoTHub Device Client Recurring Telemetry Sample")
+    print("Press Ctrl+C to exit")
+    try:
+        # Connect the client.
```

Comment:
Perhaps, like in the async sample, the contents of this try/except block (i.e. the actual function of the sample) should be factored out into a `send_recurring_telemetry` function.